Source/WebCore/dom/Node.cpp

```diff
@@ -764,8 +764,7 @@ static Node::Editability computeEditabilityFromComputedStyle(const Node& startNo
         auto* style = node->isDocumentNode() ? node->renderStyle() : const_cast<Node*>(node)->computedStyle();
         if (!style)
             continue;
-        if (style->display() == DisplayType::None)
-            continue;
+
```

Comment:
Wonder if we could just bail out with UserModify::ReadOnly in both these cases? Does it ever make sense to claim a non-renderered element is editable?